docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -1 +1,162 @@
 = Getting Started
+
+== Introduction
+
+The easiest way to get started using Solidity Hot Loader is using a https://docs.openzeppelin.com/starter-kits/2.3/[Starter Kit]. In this guide we walk through using Solidity Hot Loader with the https://docs.openzeppelin.com/starter-kits/2.3/tutorial[Tutorial Starter Kit]. The Starter Kit includes a Counter dapp, which displays the value of a Counter that can be increased/decreased. We will update the code in the smart contract to show the power of the Solidity Hot Loader.
+
+== Setup
+First we need a local blockchain to develop on. We use `ganache-cli`, so go ahead and install (if you haven't already) and then run a local blockchain using `ganache-cli`
+
+[source,console]
+----
+$ npm install -g ganache-cli
+$ ganache-cli --deterministic
+----
+
+Next we need a project directory to develop our project in. In a separate terminal create the project directory.
+
+[source,console]
+----
+$ mkdir myproject && cd myproject
+----
+
+Unpack the Tutorial Starter Kit. (This could take a little while so maybe make a cup of tea)
+
+[source,console]
+----
+$ npx @openzeppelin/cli unpack tutorial
+----
+
+Initialize the project with OpenZeppelin.
+
+[source,console]
+----
+$ npx openzeppelin init
+? Welcome to the OpenZeppelin SDK! Choose a name for your project myproject
+? Initial project version 1.0.2
+Project initialized. Write a new contract in the contracts folder and run 'openzeppelin create' to deploy it.
+----
+
+Enable the Solidity Hot Loader in `client/config/webpack.js` by setting `disabled: false` in your favorite editor. We can disable this when we need.
```

Comment:
Does this file already exist when downloading the kit, or do users need to create it?

Reply:
The file already exists, so we could cut down to just show the `disabled: false`